vaapi/vaapidisplay.cpp

```diff
@@ -320,4 +333,14 @@ DisplayPtr VaapiDisplay::create(const NativeDisplay& display)
 {
     return DisplayCache::getInstance()->createDisplay(display);
 }
+
+DisplayPtr VaapiDisplay::create(const NativeDisplay& display, VAProfile profile)
+{
+    SharedPtr<DisplayCache> cache = DisplayCache::getInstance();
+    const std::string name="hybrid";
```

Comment:
@xuguangxin, Looks like VaapiV4L2Device does not even exist.  Has someone already started working on this addition?  If so, is there a topic/fork branch somewhere?  If not, what do you envision for this VaapiV4L2Device?